### ISSUE TITLE
Writing error of IUPAC characters

### ISSUE THREAD

**Issue by EdderDaniel** [UCSC-LoweLab/tRNAscan-SE#13]
While analyzing a dataset I found a genome that contains 65 instances of an W, the IUPAC ambiguity code for either A or T. However, it appears that tRNAscan-se writes S to its fasta output, which is the IUPAC ambiguity code for either G or C.

**Comment by patriciaplchan** [comment]
I checked out the example listed at [oschwengers/bakta#150](https://github.com/oschwengers/bakta/issues/150) and ran tRNAscan-SE on the provided genome.

The tRNA-Ala is located on the "reverse" strand of LAHG01000264.1:3733-3808. The fasta output of tRNAscan-SE is 
```
>LAHG01000264.1.trna2 LAHG01000264.1:3733-3808 (-) Ala (KGC) 76 bp Sc: 67.6
GGGGGTGTAGCTCAGCTGGGAGAGCGCYTGCSTKGCASGCARGAKGTCRTCGGTTCGATCCCGTTYACCTCCACCA
```
However, the genomic sequence listed as
`genome-nt   = GGGGGTGTAGCTCAGCTGGGAGAGCGCYTGCWTKGCAWGCARGAKGTCRTCGGTTCGATCCCGTTYACCTCCACCA
`
is incorrect.

 It should be:
```
>LAHG01000264.1:3733-3808 LAHG01000264.1:3733-3808 (+)
TGGTGGAGGTRAACGGGATCGAACCGAYGACMTCYTGCWTGCMAWGCARGCGCTCTCCCAGCTGAGCTACACCCCC
```
which is the reverse complement of the output from tRNAscan-SE. 

Therefore, I am not able to reproduce the problem you are experiencing. You may want to check it out further at your end.

**Comment by patriciaplchan** [comment]
Thanks for reporting the issue. We have made the fix that is included in v2.0.11.

**Comment by oschwengers** [comment]
Hi @patriciaplchan, thanks for the fast reply and bug fix. Indeed, the term `genome-nt` wasn't well-selected. Actually, it is the nt sequence of the tRNA, and in this case already the reverse-complement of the genomic sequence. FYI, I re-edited it in the original issue https://github.com/oschwengers/bakta/issues/150 to be more precise.

> Please note that the reverse complement of W which is S is generated in the tRNAscan-SE output as expected.

For the sake of completeness, I think this is not quite correct - but it's fixed already anyways from `W->S` and `S->W` to `W->W`and `S->S`, respectively: https://github.com/UCSC-LoweLab/tRNAscan-SE/commit/fb1f8b137726e1dc4416442b084da8557510a17c

Again, thanks for the very fast reply and bug fix, and also for this great tool!
